tensorboard/components/experimental/plugin_util/BUILD

```diff
@@ -1,39 +1,18 @@
 package(default_visibility = ["//tensorboard:internal"])
 
-load("//tensorboard/defs:web.bzl", "tf_web_library")
+load("//tensorboard/defs:defs.bzl", "tf_ts_library")
 
 licenses(["notice"])  # Apache 2.0
 
-tf_web_library(
-    name = "message",
-    srcs = [
-        "message.html",
-        "message.ts",
-    ],
-    path = "/tf-plugin-util",
-)
-
-tf_web_library(
-    name = "host_internals",
-    srcs = [
-        "plugin-host-ipc.html",
-        "plugin-host-ipc.ts",
-    ],
-    path = "/tf-plugin-util",
-    deps = [
-        ":message",
-    ],
-)
-
-tf_web_library(
+tf_ts_library(
     name = "plugin_host",
     srcs = [
-        "plugin-host.html",
+        "index.ts",
+        "plugin-host-ipc.ts",
         "runs-host-impl.ts",
     ],
-    path = "/tf-plugin-util",
     deps = [
-        ":host_internals",
+        "//tensorboard/components/experimental/plugin_lib:message",
```

Comment:
WOOHOOO! Finally! Btw, I think we may want to move this to `plugin_util` instead.

Reply:
Done!  Really happy to not have to keep 2 files in sync :D